public/app/features/alerting/unified/utils/url.ts

```diff
@@ -2,5 +2,24 @@ import { config } from '@grafana/runtime';
 
 export function createUrl(path: string, queryParams?: string[][] | Record<string, string> | string | URLSearchParams) {
   const searchParams = new URLSearchParams(queryParams);
-  return `${config.appSubUrl}${path}?${searchParams.toString()}`;
+  const searchParamsString = searchParams.toString();
+
+  return `${config.appSubUrl}${path}${searchParamsString ? `?${searchParamsString}` : ''}`;
+}
+
+export function createAbsoluteUrl(
```

Comment:
This function creates an absolute URL (includes the host) but also prevents double `/` by 

1. Passing the appSubUrl and appUrl to the URL constructor
2. Setting the `pathname` on the URL object will normalize double `/`